src/cljs/athens/views/blocks.cljs

```diff
@@ -341,13 +353,13 @@
                  :content (if (clojure.string/blank? query)
                             [:div "Start Typing!"]
                             (doall
-                              [:<>
-                               (for [[i {:keys [node/title block/string block/uid]}] (map-indexed list results)]
-                                 ^{:key (str "inline-search-item" uid)}
-                                 [:div (use-style
-                                         (merge {} (when (= index i) inline-selected-search-option))
-                                         {:on-click #(prn "expand")})
-                                  (or title string)])]))}])))
+                             [:<>
+                              (for [[i {:keys [node/title block/string block/uid]}] (map-indexed list results)]
+                                ^{:key (str "inline-search-item" uid)}
+                                [:div (use-style
+                                       (merge {} (when (= index i) inline-selected-search-option))
+                                       {:on-click #(prn "expand")})
+                                 (or title string)])]))}])))
```

Comment:
Lots of indentation diffs in this PR. Do you know what's going on?

Reply:
Yes - the vscode prettifier doesn't follow the cljstyle standards, so I fixed it in the 4th commit